src/main/services/i18n/language.ts

```diff
@@ -1,6 +1,7 @@
 export const language = {
   en: 'English',
   es_ES: 'Español',
+  pt_BR: 'Português (Brasil)',
```

Comment:
I think "Português (do Brasil)" is more correct. For example, that's what appears when you select a language at https://developer.mozilla.org/.

Reply:
> I think "Português (do Brasil)" is more correct. For example, that's what appears when you select a language at https://developer.mozilla.org/.

It should be almost the same, I refer to Microsoft's.
https://docs.microsoft.com/zh-cn/locale/?target=https%3A%2F%2Fdocs.microsoft.com%2Fzh-cn%2Fdotnet%2F

Reply:
Then it doesn't matter, it's probably a matter of preference )

Reply:
This is up to you to decide. I have no opinion.